### PR TITLE
Minecraft 1.13.2 Support

### PR DESCRIPTION
**Pull request by ellisgl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (rom1504) [inline .gitignore]: Please add that in your global ignore file or in .git/info/exclude and remove it from the pr.

File: data/pc/common/protocolVersions.json [hunk 2]
```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "minecraftVersion": "1.13.2",
+    "version": 404,
+    "usesNetty": true,
+    "majorVersion": "1.13"
+  },
   {
```
Comment (rom1504) [inline data/pc/common/protocolVersions.json]: This file must contain all versions, please add the missing version (403)

**Comment by ellisgl** [comment]
I'm going to kill this PR and start over.

**Comment by rom1504** [comment]
Why ? You can rebase and (force) push on this one.

**Comment by ellisgl** [comment]
The way I read the request, you wanted individual commits for certain files. So I figured i would start over and not have the possibility of a git rebase mishap. Also I'm sleep deprived.

**Comment by mhsjlw** [comment]
He'd like to see the diff between the `protocol.json` file between versions (if you've changed it). This means just committing everything in your first commit then applying changes in the second. You can, if you'd like, just reset this PR to master then re-apply all your changes on top. Otherwise, feel free to create a new PR if that's easier.

**Pull request closed without merging**